packages/eslint-config/src/flat/configs/vue.ts

```diff
@@ -95,12 +95,17 @@ export default function vue(options: NuxtESLintConfigOptions): FlatConfig[] {
           ? {}
           : {
               // Disable Vue's default stylistic rules when stylistic is not enabled
-              'vue/max-attributes-per-line': undefined,
-              'vue/no-multi-spaces': undefined,
-              'vue/no-spaces-around-equal-signs-in-attribute': undefined,
+              'vue/html-closing-bracket-newline': undefined,
+              'vue/html-closing-bracket-spacing': undefined,
+              'vue/html-end-tags': undefined,
```

Comment:
I am not sure if end-tags should be count as stylistic rule: https://eslint.vuejs.org/rules/html-end-tags